workers/unity/Assets/Playground/Scripts/Player/ProcessLaunchCommandSystem.cs

```diff
@@ -18,8 +18,7 @@ private struct LaunchCommandData
             [ReadOnly] public EntityArray Entity;
             public ComponentDataArray<SpatialOSLauncher> Launcher;
 
-            [ReadOnly]
-            public ComponentArray<CommandRequests<Generated.Playground.Launcher.LaunchEntity.Request>> CommandRequests;
+            [ReadOnly] public ComponentArray<CommandRequests<Generated.Playground.Launcher.LaunchEntity.Request>> CommandRequests;
```

Comment:
Please revert this (if you merge with master you'll have the linting fixes that stop this flip flopping)